suite-native/firmware/src/screens/FirmwareUpdateInProgressScreen.tsx

```diff
@@ -81,6 +88,10 @@ export const FirmwareUpdateInProgressScreen = () => {
         navigation.goBack();
     }, [dispatch, navigation]);
 
+    const handleCancel = useCallback(() => {
+        navigation.goBack();
```

Comment:
Do we need to cancel the device interaction as well? 🤔

Reply:
I think we do not, if there is error it means connect call ended with that error so there is nothing to cancel.